frontend/src/lib/components/Drawer.tsx

```diff
@@ -16,5 +17,5 @@ export function Drawer(props: PropsWithChildren<DrawerProps>): JSX.Element {
         }
     }, [visible])
 
-    return <AntDrawer {...props} />
+    return <AntDrawer {...props} zIndex={styles.zDrawer} />
```

Comment:
Wouldn't it be better if we did this via classNames

Reply:
Looks like Ant's component overrides it in the JS style, that's why we need to use their prop.

Reply:
!important might still win here :) But I get the point now